### ISSUE TITLE
Tabbing out of the game through cmd tab or ESC crashes my client.

### ISSUE THREAD

**Issue by vxxsen** [Wurst-Imperium/Wurst7#1025]
### Pre-reporting checklist

- [x] I have searched existing issues and didn't find any previous reports of this bug.
- [x] This bug still happens in the latest version of Wurst. I've made sure my installation is up to date.
- [ ] I have tried removing all other mods and the bug still happens.

### Description

Whenever I am in a server and I try to tab out, my game instantly goes to a save screen and crashes. When I am in the server selector or title screen this doesn't happen, only when I am playing on a server.  I do have other mods on, but through removing a large portion of them (about half) I came to the conclusion that this was probably a wurst thing.

### Repro steps

1. Hit ESC
2. Hit cmd + tab
3. Quilt Loader
4. Macbook Air M2

### Wurst Client version

7.43.1

### Minecraft version

1.21

### Crash report (if applicable)

```plaintext
Mod Table Version: 2
Plugin Types: {quilted_fabric_loader=[Fabric], quilt_loader=[Quilt, Builtin]}
[17:23:01] [QuiltClassPath ZipScanner#1/INFO]: Took 46248us to scan /Users/{COMPUTER_USERNAME}/Library/Application Support/com.modrinth.theseus/profiles/Waddon1_s Performance Modpack 1.21/.cache/quilt_loader/transform-cache-client/transformed-mod-minecraft-v1.21.jar /
[17:23:01] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/Users/{COMPUTER_USERNAME}/Library/Application%20Support/com.modrinth.theseus/meta/libraries/net/fabricmc/sponge-mixin/0.13.3+mixin.0.8.5/sponge-mixin-0.13.3+mixin.0.8.5.jar Service=Knot/Quilt Env=CLIENT
[17:23:01] [main/INFO]: Compatibility level set to JAVA_16
[17:23:01] [main/INFO]: Compatibility level set to JAVA_17
[17:23:01] [main/INFO]: Compatibility level set to JAVA_21
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.fixes.general.threading_issues.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.base.mixin
[17:23:01] [main/INFO]: Global Executor Parallelism: 1 configured, 1 evaluated, 1 default evaluated
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.client.uncapvd.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.fixes.chunkio.threading_issues.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.opts.worldgen.general.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.notickvd.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.opts.chunkio.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.threading.lighting.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.opts.chunk_access.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.threading.chunkio.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.fixes.worldgen.threading_issues.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.fixes.worldgen.vanilla_bugs.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.opts.allocs.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.opts.worldgen.vanilla.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.server.utils.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.opts.math.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.opts.scheduling.mixin
[17:23:01] [main/INFO]: Initializing com.ishland.c2me.threading.worldgen.mixin
[17:23:01] [main/INFO]: Disabling com.ishland.c2me.threading.worldgen.mixin
[17:23:01] [main/INFO]: Loaded configuration file for Sodium: 42 options available, 3 override(s) found
[17:23:01] [main/INFO]: Loaded configuration file for Sodium Extra: 35 options available, 0 override(s) found
[17:23:01] [main/WARN]: Mod 'c2me' attempted to override option 'mixin.world.player_chunk_tick', which doesn't exist, ignoring
[17:23:01] [main/INFO]: Loaded configuration file for Lithium: 138 options available, 1 override(s) found
[17:23:01] [main/WARN]: Error loading class: net/caffeinemc/sodium/config/user/UserConfigCategories (java.lang.ClassNotFoundException: net/caffeinemc/sodium/config/user/UserConfigCategories)
[17:23:01] [main/WARN]: Error loading class: ca/spottedleaf/starlight/common/thread/SchedulingUtil (java.lang.ClassNotFoundException: ca/spottedleaf/starlight/common/thread/SchedulingUtil)
[17:23:01] [main/WARN]: Error loading class: com/ishland/c2me/rewrites/chunk_serializer/common/ChunkDataSerializer (java.lang.ClassNotFoundException: com/ishland/c2me/rewrites/chunk_serializer/common/ChunkDataSerializer)
[17:23:01] [main/WARN]: Force-disabling mixin 'features.render.entity.CuboidMixin' as rule 'mixin.features.render.entity' (added by mods [iris]) disables it and children
[17:23:01] [main/WARN]: Force-disabling mixin 'features.render.entity.ModelPartMixin' as rule 'mixin.features.render.entity' (added by mods [iris]) disables it and children
[17:23:01] [main/WARN]: Force-disabling mixin 'features.render.entity.cull.EntityRendererMixin' as rule 'mixin.features.render.entity' (added by mods [iris]) disables it and children
[17:23:01] [main/WARN]: Force-disabling mixin 'features.render.entity.shadows.EntityRenderDispatcherMixin' as rule 'mixin.features.render.entity' (added by mods [iris]) disables it and children
[17:23:01] [main/WARN]: Force-disabling mixin 'features.render.gui.font.GlyphRendererMixin' as rule 'mixin.features.render.gui.font' (added by mods [iris]) disables it and children
[17:23:01] [main/WARN]: Force-disabling mixin 'features.render.world.sky.BackgroundRendererMixin' as rule 'mixin.features.render.world.sky' (added by mods [iris]) disables it and children
[17:23:01] [main/WARN]: Force-disabling mixin 'features.render.world.sky.ClientWorldMixin' as rule 'mixin.features.render.world.sky' (added by mods [iris]) disables it and children
[17:23:01] [main/WARN]: Force-disabling mixin 'features.render.world.sky.WorldRendererMixin' as rule 'mixin.features.render.world.sky' (added by mods [iris]) disables it and children
[17:23:01] [main/WARN]: Error loading class: me/jellysquid/mods/sodium/client/render/occlusion/BlockOcclusionCache (java.lang.ClassNotFoundException: me/jellysquid/mods/sodium/client/render/occlusion/BlockOcclusionCache)
[17:23:01] [main/WARN]: Error loading class: me/jellysquid/mods/sodium/client/render/pipeline/FluidRenderer (java.lang.ClassNotFoundException: me/jellysquid/mods/sodium/client/render/pipeline/FluidRenderer)
[17:23:01] [main/WARN]: Force-disabling mixin 'alloc.blockstate.StateMixin' as rule 'mixin.alloc.blockstate' (added by mods [ferritecore]) disables it and children
[17:23:01] [main/WARN]: Error loading class: net/caffeinemc/sodium/config/user/UserConfigCategories (java.lang.ClassNotFoundException: net/caffeinemc/sodium/config/user/UserConfigCategories)
[17:23:01] [main/WARN]: Error loading class: net/caffeinemc/sodium/config/user/UserConfigCategories (java.lang.ClassNotFoundException: net/caffeinemc/sodium/config/user/UserConfigCategories)
[17:23:01] [main/WARN]: Error loading class: me/jellysquid/mods/sodium/client/render/occlusion/BlockOcclusionCache (java.lang.ClassNotFoundException: me/jellysquid/mods/sodium/client/render/occlusion/BlockOcclusionCache)
[17:23:01] [main/WARN]: Error loading class: me/jellysquid/mods/sodium/client/render/occlusion/BlockOcclusionCache (java.lang.ClassNotFoundException: me/jellysquid/mods/sodium/client/render/occlusion/BlockOcclusionCache)
[17:23:01] [main/WARN]: Error loading class: me/jellysquid/mods/sodium/client/render/pipeline/FluidRenderer (java.lang.ClassNotFoundException: me/jellysquid/mods/sodium/client/render/pipeline/FluidRenderer)
[17:23:01] [main/WARN]: Error loading class: me/jellysquid/mods/sodium/client/render/pipeline/FluidRenderer (java.lang.ClassNotFoundException: me/jellysquid/mods/sodium/client/render/pipeline/FluidRenderer)
[17:23:01] [main/INFO]: Searching for graphics cards...
[17:23:02] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.3.5).
[17:23:02] [Datafixer Bootstrap/INFO]: 226 Datafixer optimizations took 136 milliseconds
[17:23:07] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[17:23:07] [Render thread/INFO]: Setting user: {MINECRAFT_USERNAME}
[17:23:07] [Render thread/INFO]: [STDOUT]: Starting Wurst Client...
[17:23:07] [Render thread/INFO]: Checking mod updates...
[17:23:07] [Render thread/INFO]: Loading Xaero's Minimap - Stage 1/2
[17:23:07] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[17:23:07] [Render thread/INFO]: [BetterF3] Loading...
[17:23:07] [Render thread/INFO]: [BetterF3] All done!
[17:23:07] [Render thread/INFO]: Dynamic FPS 3.4.3 active on Fabric!
[17:23:07] [Render thread/INFO]: Registering S2C receiver with id architectury:spawn_entity_packet
[17:23:07] [Render thread/INFO]: Loading Xaero's World Map - Stage 1/2
[17:23:07] [Worker-Main-1/INFO]: Update available for 'modmenu@11.0.0', (-> 11.0.1)
[17:23:08] [Render thread/INFO]: Backend library: LWJGL version 3.3.3+5
[17:23:08] [ForkJoinPool.commonPool-worker-4/WARN]: [Iris Update Check] This version doesn't have an update index, skipping.
[17:23:08] [Render thread/INFO]: Initializing ImmediatelyFast 1.2.18+1.21 on Apple M2 (Apple) with OpenGL 4.1 Metal - 88
[17:23:08] [Render thread/WARN]: Force disabling fast buffer upload optimization due to unsupported GPU
[17:23:08] [Render thread/INFO]: Found Iris/Oculus 1.7.1+mc1.21. Enabling compatibility.
[17:23:09] [Render thread/INFO]: Debug functionality is disabled.
[17:23:09] [Render thread/INFO]: DSA support not detected.
[17:23:09] [Render thread/INFO]: Shaders are disabled because no valid shaderpack is selected
[17:23:09] [Render thread/INFO]: Hardware information:
[17:23:09] [Render thread/INFO]: CPU: 8x Apple M2
[17:23:09] [Render thread/INFO]: GPU: Apple M2 (Supports OpenGL 4.1 Metal - 88)
[17:23:09] [Render thread/INFO]: OS: Mac OS X (14.4.1)
[17:23:09] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric, betterf3, c2me, caffeineconfig, cesium, cloth-config, dynamic_fps, entityculling, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, immediatelyfast, iris, lithium, modmenu, nochatreports, quilt_loader, reeses-sodium-options, sodium, sodium-extra, wurst, xaerominimap, xaeroworldmap
[17:23:09] [Worker-Main-6/INFO]: Found unifont_all_no_pua-15.1.05.hex, loading
[17:23:09] [Worker-Main-1/INFO]: Found unifont_jp_patch-15.1.05.hex, loading
[17:23:09] [Render thread/INFO]: Loading Xaero's World Map - Stage 2/2
[17:23:09] [Render thread/INFO]: New world map region cache hash code: 919872734
[17:23:09] [Render thread/INFO]: Registered player tracker system: map_synced
[17:23:09] [Render thread/INFO]: Xaero's WorldMap Mod: Xaero's minimap found!
[17:23:09] [Render thread/INFO]: Registered player tracker system: minimap_synced
[17:23:09] [Render thread/INFO]: No Optifine!
[17:23:09] [Render thread/INFO]: Xaero's World Map: No Vivecraft!
[17:23:09] [Render thread/INFO]: Xaero's World Map: Iris found!
[17:23:09] [Render thread/INFO]: Loading Xaero's Minimap - Stage 2/2
[17:23:10] [Render thread/INFO]: Registered player tracker system: minimap_synced
[17:23:10] [Render thread/INFO]: Xaero's Minimap: World Map found!
[17:23:10] [Render thread/INFO]: No Optifine!
[17:23:10] [Render thread/INFO]: Xaero's Minimap: No Vivecraft!
[17:23:10] [Render thread/INFO]: Xaero's Minimap: Iris found!
[17:23:10] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[17:23:10] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[17:23:10] [Render thread/INFO]: OpenAL initialized on device Aiden’s Beats Solo³
[17:23:10] [Render thread/INFO]: Sound engine started
[17:23:10] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[17:23:10] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[17:23:10] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[17:23:10] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[17:23:10] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[17:23:10] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[17:23:10] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[17:23:10] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[17:23:10] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[17:23:10] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[17:23:10] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/paintings.png-atlas
[17:23:10] [Render thread/INFO]: Created: 256x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[17:23:10] [Render thread/INFO]: Created: 64x64x0 minecraft:textures/atlas/map_decorations.png-atlas
[17:23:10] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/gui.png-atlas
[17:23:10] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[17:23:11] [Render thread/INFO]: Successfully reloaded the minimap shaders!
[17:23:11] [Render thread/INFO]: Successfully reloaded the world map shaders!
[17:23:11] [Render thread/INFO]: Creating pipeline for dimension minecraft:overworld
[17:23:14] [Render thread/INFO]: Connecting to frozendischarge.aternos.me, 25565
[17:23:15] [Render thread/INFO]: New Xaero hud session initialized!
[17:23:16] [Render thread/INFO]: New world map session initialized!
[17:23:16] [Render thread/INFO]: Enforcing safe world random access
[17:23:16] [Render thread/INFO]: Reloading pipeline on dimension change: minecraft:overworld => minecraft:overworld
[17:23:16] [Render thread/INFO]: Destroying pipeline minecraft:overworld
[17:23:16] [Render thread/INFO]: Creating pipeline for dimension minecraft:overworld
[17:23:16] [Render thread/WARN]: Error loading class: me/jellysquid/mods/sodium/client/render/occlusion/BlockOcclusionCache (java.lang.ClassNotFoundException: me/jellysquid/mods/sodium/client/render/occlusion/BlockOcclusionCache)
[17:23:16] [Render thread/WARN]: Error loading class: me/jellysquid/mods/sodium/client/render/pipeline/FluidRenderer (java.lang.ClassNotFoundException: me/jellysquid/mods/sodium/client/render/pipeline/FluidRenderer)
[17:23:16] [Render thread/INFO]: Started 2 worker threads
[17:23:16] [Render thread/INFO]: [CHAT] {MINECRAFT_USERNAME} joined the game
[17:23:16] [Render thread/INFO]: Loaded 323 advancements
[17:23:16] [Render thread/INFO]: Stopping worker threads
[17:23:16] [Render thread/INFO]: Started 2 worker threads
[17:23:16] [Render thread/INFO]: Reloading entity icon resources...
[17:23:16] [Render thread/INFO]: Reloaded entity icon resources!
[17:23:16] [Render thread/INFO]: [STDOUT]: Brightness started at 0.5
[17:23:17] [WurstUpdater/INFO]: [STDOUT]: [Updater] Current version: 7.43.1
[17:23:17] [WurstUpdater/INFO]: [STDOUT]: [Updater] Latest version: 7.43.1
[17:23:17] [Render thread/ERROR]: Reported exception thrown!
net.minecraft.class_148: Someone deleted the Feedback button!
	at net.minecraft.class_433.addWurstOptionsButton(class_433.java:1110) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_433.handler$baf000$wurst$onInitWidgets(class_433.java:1053) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_433.method_20543(class_433.java:121) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_433.method_25426(class_433.java:68) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_437.method_25423(class_437.java:309) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_310.method_1507(class_310.java:1143) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_310.method_20539(class_310.java:1721) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_309.method_1466(class_309.java:451) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_309.method_22678(class_309.java:499) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_1255.execute(class_1255.java:108) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_309.method_22676(class_309.java:499) ~[transformed-mod-minecraft-v1.21.jar:?]
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:44) ~[lwjgl-glfw-3.3.3.jar:build 5]
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.3.3.jar:build 5]
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3438) ~[lwjgl-glfw-3.3.3.jar:build 5]
	at com.mojang.blaze3d.systems.RenderSystem.pollEvents(RenderSystem.java:150) ~[transformed-mod-minecraft-v1.21.jar:?]
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:168) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_1041.method_15998(class_1041.java:303) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1307) ~[transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:882) [transformed-mod-minecraft-v1.21.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:256) [transformed-mod-minecraft-v1.21.jar:?]
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:557) [quilt-loader-0.26.1-beta.1.jar:?]
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:84) [quilt-loader-0.26.1-beta.1.jar:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:28) [quilt-loader-0.26.1-beta.1.jar:?]
Caused by: java.lang.IllegalStateException
	at net.minecraft.class_433.addWurstOptionsButton(class_433.java:1109) ~[transformed-mod-minecraft-v1.21.jar:?]
	... 24 more
[17:23:17] [Render thread/INFO]: Xaero hud session finalized.
[17:23:17] [Render thread/INFO]: Finalizing world map session...
[17:23:17] [Thread-5/INFO]: World map force-cleaned!
[17:23:17] [Render thread/INFO]: World map session finalized.
[17:23:17] [Render thread/INFO]: Stopping worker threads
```

**Comment by vxxsen** [comment]
Ok loading a basic quilt client with quilt and fabric does not have this issue, saw the last issue with the modmenu confliction. Will see if this fixes.

**Comment by vxxsen** [comment]
Removing Mod Menu fixes problem.

**Comment by jeremyg39** [comment]
Ok noted— will fix soon. Thx

**Comment by Alexander01998** [comment]
@jeremyg39 Who are you and why are you commenting on this? 😅

Anyway, I've pushed a fix for this issue in a2e2ee1bde7377a27a1a9194fc59961b0bc444df, the new version is just not generally available yet. For now, downgrading or removing ModMenu can be used as a workaround, or changing ModMenu's "Game menu: Replace" setting to either "Game menu: Insert" or "Game menu: Icon".
